src/test/java/com/epam/automation/careerTest.java

```diff
@@ -1,101 +0,0 @@
-package com.epam.automation;
-
-import com.epam.automation.enums.Menu;
-import org.openqa.selenium.By;
-import org.testng.annotations.Test;
-
-public class careerTest extends inside {
-
-    @Test
-    public void navigationTest() {
-        navigation(Menu.WHAT_WE_DO);
-        navigation(Menu.ABOUT);
-        navigation(Menu.HOW_WE_DO_IT);
-        navigation(Menu.INSIGHTS);
-        navigation(Menu.OUR_WORK);
-        navigation(Menu.CAREER);
-    }
-
-    @Test
-    public void searchButtonTest() {
```

Comment:
Test should be more self explaining. I don't really understand what this tests and how